### PR TITLE
feat: add governance docs

### PR DESCRIPTION
**Pull request by csatib02**
- Adds written open Governance, requested on the [CNCF project onboarding checklist](https://github.com/cncf/sandbox/issues/142)

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+Bank-Vaults follows a consensus-seeking decision-making process. For most decisions, we strive for agreement among maintainers. If consensus cannot be reached, the project lead (or a designated decision-maker) will make the final call.
+
+## Contributing
+
+We welcome contributions from everyone. To get started:
+
+1. Fork the repository
+2. Create a new branch for your changes
+3. Make your changes and commit them with clear, descriptive messages
```
Comment (akijakya) [inline GOVERNANCE.md]:
```suggestion
3. Make your changes and commit them with clear, descriptive messages, signed off by the author
```

File: GOVERNANCE.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+### Users
+
+Users are the foundation of the Bank-Vaults community. They utilize the project in their environments and provide valuable feedback. We encourage users to:
+
+- Share their experiences with Bank-Vaults
+- Report issues and suggest improvements
+- Participate in discussions on our communication channels
+- Contribute to documentation by reporting unclear or outdated sections
+
```
Comment (akijakya) [inline GOVERNANCE.md:16]: Maybe we should add a line here about users of Bank-Vaults are encouraged to add themselves to our [Adopters list](https://github.com/bank-vaults/bank-vaults/blob/main/ADOPTERS.md)?

**Review by akijakya — COMMENTED** [review]
Good job, thanks! 🔥

**Review by sagikazarmark — APPROVED** [review]
Good job!

Generally, it's better to adopt a simple governance process that accurately depicts the status quo (with the potential to iterate) than introducing complicated new processes and this one does just that, so 👍 .

**Review by akijakya — APPROVED** [review]
LGTM 🚀

**Review by lgecse — APPROVED** [review]

**Review by ramizpolic — APPROVED** [review]

**Pull request merged**